modules/update-conditional-updates.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-cli.adoc
+// * updating/understanding-upgrade-channels-releases.adoc
+
+[id="update-conditional-upgrade-path{context}"]
+= Updating to a conditional upgrade path
```

Comment:
Changing to `Updating to a conditional update` is better?

Reply:
`Updating to a conditional update` is a bit repetitive. Changing to `Updating along a conditional upgrade path`.